src/components/Contentful/Hero.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import heroContent from '../../../test/fixtures/contentful/hero'
+import Contentful from '.'
+
+describe('Hero', () => {
+  describe('when content provided with no image', () => {
+    const contentful = mount(<Contentful content={[heroContent()]} />)
```

Comment:
Shallow mounting is better for unit testing I believe as it doesn't try to render any components that are called inside the one you are testing.
Think mount is typically preferred for integration tests.

Reply:
This test would need the child components to be rendered though as it's the hero component inside the Contentful component that is ultimately the one being tested